### PR TITLE
Fix bugs in calc interpolation migrator

### PR DESCRIPTION
**Pull request by pamelalozano16**
Migration for more than one interpolation or expressions in a calc function parameter.

### CODE REVIEW

File: test/migrators/migrator_dart_test.dart [hunk 1]
```diff
@@ -12,4 +12,5 @@ main() {
   testMigrator("module");
   testMigrator("namespace");
   testMigrator("strict_unary");
+  testMigrator("calc_interpolation");
```
Comment (jathak) [inline test/migrators/migrator_dart_test.dart:15]: Sort this

**Review by jathak — APPROVED** [review]
Can you bump the version to 1.8.1 and add a changelog entry?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.8.1
+
+* Migration for more than one interpolation or expressions in a calc function parameter.
```
Comment (jathak) [inline CHANGELOG.md:3]: Can you add a subheading here to note this is for the calc interpolation migrator? (same as the 1.8.0 changelog entry)

Comment (pamelalozano16) [reply CHANGELOG.md:5]: Done!

**Pull request merged**